lib/Extractor/Candidates.cpp

```diff
@@ -158,16 +158,18 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
   unsigned Width = DL.getTypeSizeInBits(V->getType());
   APInt KnownZero(Width, 0, false), KnownOne(Width, 0, false);
   bool NonZero = false, NonNegative = false, PowOfTwo = false;
+  unsigned NumSignBits = 1;
   if (HarvestKnownBits)
```

Comment:
plz change this to HarvestDataflow or similar, since it is doing more than known bits